src/views/completed.pug

```diff
@@ -1,9 +1,10 @@
 doctype html
 head
+  link(rel='stylesheet', type='text/css', href='style.css')
 body
+  h2 Todays completed tasks
   ul
     each task in tasks
-      li= task
-    
-    
-    
+      li= task.description
+      button(class='completed') Done
+      script(src="/public/index.js")
```

Comment:
You would actually link this script in the header or footer of the html doc. To make the button functional you will need to allow it to somehow have access to the id of each item, then it could be a link that calls a route, or use your index.js file in the public folder to add an event listener but regardless your function requires you pass it the id of a task to be able to mark it completed.